shell/platform/darwin/ios/flutter_unittests.mm

```diff
@@ -0,0 +1,19 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+
+#include "flutter/testing/testing.h"
+
+namespace flutter_tests {
+
+TEST(SmokeTest, Success) {
+  EXPECT_EQ(1, 1);
+}
+
+TEST(SmokeTest, Fail) {
```

Comment:
This requires a different harness.

Reply:
Right now this test isn't run and nothing is tested.  I wanted to see if we could somehow get the exit code from this or what log scraping might make sense for CI.